src/qt/addresstablemodel.cpp

```diff
@@ -89,7 +89,7 @@ class AddressTablePriv
         // qLowerBound() and qUpperBound() require our cachedAddressTable list to be sorted in asc order
```

Comment:
Could the comment be fixed by removing/replacement mentions of `qLowerBound()` and `qUpperBound()`?

Reply:
Agree, let's keep the comment but with the new method names

Reply:
Have reinstated the comment, using the new method names.